Neos.ContentRepository.Core/Classes/Factory/ContentRepositoryFactory.php

```diff
@@ -100,7 +101,7 @@ public function getOrBuild(): ContentRepository
                 $this->projectionFactoryDependencies->interDimensionalVariationGraph,
                 $this->projectionFactoryDependencies->contentDimensionSource,
                 $this->userIdProvider,
-                $this->clock,
+                $this->clock
```

Comment:
Please leave those trailing commas in so we don't create multi-line-diffs when adding new arguments
```suggestion
                $this->clock,
```